src/v/cluster/members_backend.cc

```diff
@@ -226,6 +226,36 @@ ss::future<> members_backend::calculate_reallocations(update_meta& meta) {
         co_return;
     }
 }
+
+void members_backend::reallocations_for_even_partition_count(
+  members_backend::update_meta& meta, partition_allocation_domain domain) {
+    calculate_reallocations_for_even_partition_count(meta, domain);
```

Comment:
Nit: naming here is a bit confusing. We have `reallocations_for_even_partition_count` and `calculate_reallocations_for_even_partition_count` that are kind of the same (both calculate) but one calls the other... It would be cool if they had more distinct names.